### PR TITLE
Creating a brand new page with taurus specific information

### PR DESCRIPTION
**Pull request by randywessels**
Creating a brand new page with taurus specific information

### CODE REVIEW

**Review by jim-counter — CHANGES_REQUESTED** [review]
I think we can improve the amount of shared content between the mainnet and taurus docs here. I also noticed that the systemd generator displays `--chain mainnet`.

File: docs/develop/intro.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ To start using Nova, refer to our [Guides and Manuals](/develop/nova/introductio
 - wss://rpc-0.gemini-3h.subspace.network/ws
 - wss://rpc-1.gemini-3h.subspace.network/ws
 - wss://rpc.taurus.subspace.foundation/ws
+- wss://rpc.mainnet.subspace.foundation/ws
```
[the remark this answers is not in the document]
Comment (EmilFattakhov) [reply docs/develop/intro.md:65]: Weird that they're on different domains 😢

File: docs/farming-&-staking/farming/common-problems.mdx [hunk 2]
```diff
@@ -43,7 +43,7 @@ These errors and warnings are tied to specific components. In this section, we w
 
 This should resolve on its own.
 
-<small>Last Confirmed: taurus-2024-oct-24</small>
+<small>Last Confirmed: mainnet-2024-nov-06-24</small>
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/common-problems.mdx:46]:
```suggestion
<small>Last Confirmed: mainnet-2024-nov-06</small>
```

File: docs/farming-&-staking/farming/common-problems.mdx [hunk 3]
```diff
@@ -52,7 +52,7 @@ This should resolve on its own.
 
 🟠 This warning message should not affect your rewards or the operation of your node. It's just a log message indicating the normal operation of the network.
 
-<small>Last Confirmed: taurus-2024-oct-24</small>
+<small>Last Confirmed: mainnet-2024-nov-06-24</small>
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/common-problems.mdx:55]:
```suggestion
<small>Last Confirmed: mainnet-2024-nov-06</small>
```

File: docs/farming-&-staking/farming/common-problems.mdx [hunk 4]
```diff
@@ -68,7 +68,7 @@ This should resolve on its own.
 
 The drive where you are storing your plot is out of space. Reduce the plot size and restart the farmer. Additionally, ensure that you are saving the plot to the correct drive and that it is properly mounted.
 
-<small>Last Confirmed: taurus-2024-oct-24</small>
+<small>Last Confirmed: mainnet-2024-nov-06-24</small>
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/common-problems.mdx:71]:
```suggestion
<small>Last Confirmed: mainnet-2024-nov-06</small>
```

File: docs/farming-&-staking/farming/common-problems.mdx [hunk 5]
```diff
@@ -82,7 +82,7 @@ The drive where you are storing your plot is out of space. Reduce the plot size
 
 🟠 This is a known issue that is actively being investigated. To help mitigate these errors, run your NATS component on the same system as your Controller, Cache, and Plotter components.
 
-<small>Last Confirmed: taurus-2024-oct-24</small>
+<small>Last Confirmed: mainnet-2024-nov-06-24</small>
```
Comment (EmilFattakhov) [inline docs/farming-&-staking/farming/common-problems.mdx:85]:
```suggestion
<small>Last Confirmed: mainnet-2024-nov-06</small>
```

File: src/theme/DocSidebar/index.js [hunk 6]
```diff
@@ -8,7 +8,7 @@ export default function DocSidebarWrapper(props) {
       <div className="sidebar-recommendation-label">
         ✨ Denotes recommended application <br />
         ♊ Denotes availability on Gemini-3h testnet <br />
-        ♉ Denotes availability on Taurus testnet
+        ♉ Denotes availability on Taurus testnet / Mainnet
```
Comment (EmilFattakhov) [inline src/theme/DocSidebar/index.js:11]: No, this won't be the case, mainnet will use another emoji

File: docs/develop/intro.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (EmilFattakhov) [reply docs/develop/intro.md:65]: Okay, clarified that both are legit, one from the foundation, one from the labs

Should we include one of these too? 

```
`wss://rpc-0.mainnet.subspace.network/ws`
`wss://rpc-1.mainnet.subspace.network/ws`
`wss://rpc-0.mainnet.autonomys.xyz/ws`
`wss://rpc-1.mainnet.autonomys.xyz/ws`
```

[the remark this answers is not in the document]
Comment (randywessels) [reply docs/develop/intro.md:65]: I am thinking that if there is a foundation version of something that is what we should use. I am seeing comments in Slack about emphasizing things like the foundation is the one launching the chain. I think we should add both rpc-0 and rpc-1 though.  Just not sure if taurus has that as well or not. Also, maybe we should remove the gemini-3h ones?

File: src/theme/DocSidebar/index.js (hunk 6 above)
Comment (randywessels) [reply src/theme/DocSidebar/index.js:11]: I figured as much, but we wanted to get this PR out as soon as possible. It also serves as the updates for today's mainnet release.

File: docs/farming-&-staking/farming/advanced-cli/taurus-platforms/_systemd.mdx [hunk 7]
```diff
@@ -0,0 +1,208 @@
...
+	<CodeBlock language="shell-session">
+	{`wget -O ~/.local/bin/subspace-node https://github.com/autonomys/subspace/releases/download/taurus-2024-nov-05/subspace-node-ubuntu-aarch64-taurus-2024-nov-05`}
+	</CodeBlock>
+	Farmer:
+	<CodeBlock language="shell-session">
+	{`wget -O ~/.local/bin/subspace-farmer https://github.com/autonomys/subspace/releases/download/taurus-2024-nov-05/subspace-farmer-ubuntu-aarch64-taurus-2024-nov-05`}
+	</CodeBlock>
+</details>
+
+By default, these commands download the executable files to the `~/.local/bin` directory.
```
[the remark this answers is not in the document]
Comment (randywessels) [reply docs/farming-&-staking/farming/advanced-cli/taurus-platforms/_systemd.mdx:81]: I was originally going to do that, but then not only would the network would show mainnet, but also the executable links wouldn't be for taurus.

**Pull request merged**